Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/light_rifle.yml

```
@@ -0,0 +1,79 @@
+- type: entity
+  id: SunriseCartridgeLightRiflePractice
+  name: cartridge (7,62mm practice)
+  parent: SunriseBaseCartridgeLightRifle
+  description: A classic intermediate cartridge 7,62x39mm used by many combat rifles and LMGs. Standard kinetic ammunition is common and useful in most situations. Chalk ammunition is generally non-harmful, used for practice.
+  components:
+  - type: HitScanCartridgeAmmo
+    proto: BulletLightRiflePractice
+  - type: Sprite
+    layers:
+      - state: base
+        map: [ "enum.AmmoVisualLayers.Base" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)